### ISSUE TITLE
Storyboard issue with 1.1.1

### ISSUE THREAD

**Issue by AFcgi** [AliSoftware/SwiftGen#139]
In <code> initialViewController</code>-method is missing a return value.
```
extension StoryboardSceneType {
  static func storyboard() -> UIStoryboard {
    return UIStoryboard(name: self.storyboardName, bundle: nil)
  }

  static func initialViewController() -> UIViewController {
    guard let vc = storyboard().instantiateInitialViewController() else {
      fatalError("Failed to instantiate initialViewController for \(self.storyboardName)")
    }
  }
}
```

**Comment by ikesyo** [comment]
The fix is already submitted in #138.

**Comment by AliSoftware** [comment]
Good catch. closing this issue as someone already suggested a PR #128 to fix though.
